src/firefly/js/visualize/iv/ImageViewer.jsx

```diff
@@ -61,7 +61,7 @@ function getStoreState(plotId, oldState) {
 
 const TEN_SECONDS= 10000;
 
-export const ImageViewer= memo( ({showWhenExpanded=false, plotId}) => {
+export const ImageViewer= memo( ({showWhenExpanded=false, plotId, makeToolbar}) => {
```

Comment:
Normally, this is a good reason to create slots.  `makeToolbar` would be slotProps.component.

Reply:
yes, I plan to do that in the future.